CHANGELOG.md

```diff
@@ -1,8 +1,126 @@
+- Update zune-jpeg to fix crash on bad JPEGs [#7952](https://github.com/rerun-io/rerun/pull/7952)
+- Consistent open/import/log_file behaviors in all common scenarios [#7966](https://github.com/rerun-io/rerun/pull/7966)
+- ChunkStore: fix row-id computation when removing dangling static chunks [#8020](https://github.com/rerun-io/rerun/pull/8020)
+- `EntityTree`: only check for entity deletions when necessary [#8103](https://github.com/rerun-io/rerun/pull/8103)
+
+#### 🌁 Viewer improvements
+- A Rerun Viewer session now matches 1:1 to a Rerun TCP server [#6951](https://github.com/rerun-io/rerun/pull/6951) (thanks [@petertheprocess](https://github.com/petertheprocess)!)
+- Implement support for in-place drag-n-drop [#7880](https://github.com/rerun-io/rerun/pull/7880)
+- Implement `Menu > Import` and associated command [#7882](https://github.com/rerun-io/rerun/pull/7882)
+- Expose additional information about decoded frames in the viewer [#7932](https://github.com/rerun-io/rerun/pull/7932)
+- Update crates, including `rfd` for better file dialogs [#7953](https://github.com/rerun-io/rerun/pull/7953)
+- Line strips are no longer a disconnected series of quads [#8065](https://github.com/rerun-io/rerun/pull/8065)
+
+#### 🚀 Performance improvements
+- Don't keep around additional CPU copy of loaded mesh files [#7824](https://github.com/rerun-io/rerun/pull/7824)
+- Make mp4 parsing **a lot** faster & tremendously lower memory overhead [#7860](https://github.com/rerun-io/rerun/pull/7860)
+- Fix slow receive when using native WebSocket [#7875](https://github.com/rerun-io/rerun/pull/7875)
+- Implement support for fully asynchronous `QueryHandle`s [#7964](https://github.com/rerun-io/rerun/pull/7964)
+
+#### 🧑‍🏫 Examples
+- Fix Rust DNA sample writing to a temporary file [#7827](https://github.com/rerun-io/rerun/pull/7827)
+- Add `ml_depth_pro` example [#7832](https://github.com/rerun-io/rerun/pull/7832) (thanks [@oxkitsune](https://github.com/oxkitsune)!)
+- Add map view to nuscenes python example [#8034](https://github.com/rerun-io/rerun/pull/8034) (thanks [@tfoldi](https://github.com/tfoldi)!)
+- Add an example to display OpenStreetMap-sourced data on the map view [#8044](https://github.com/rerun-io/rerun/pull/8044)
+
+#### 📚 Docs
+- Clarify viewport documentation and reference the type list for view classes [#7826](https://github.com/rerun-io/rerun/pull/7826)
+- Finish dataframe reference page [#7865](https://github.com/rerun-io/rerun/pull/7865)
+- Docs: static data [#7856](https://github.com/rerun-io/rerun/pull/7856)
+- Docs: concepts > recordings [#7896](https://github.com/rerun-io/rerun/pull/7896)
+- Docs: "How-to: reuse blueprints across languages" [#7886](https://github.com/rerun-io/rerun/pull/7886)
+- Docs: application model part 1: native workflows [#7905](https://github.com/rerun-io/rerun/pull/7905)
+- Document arrow datatypes [#7986](https://github.com/rerun-io/rerun/pull/7986)
```

Comment:
This might we worth highlighting

Reply:
35755b44063d7bfc9e583c6ca40750e5af9dd99b